train.py

```diff
@@ -0,0 +1,201 @@
+# pip install timm wandb tqdm datasets
```

Comment:
let's put this in the benchmarks folder

Reply:
Moved to `benchmarks/benchmark_adam_8bit.py`